README.md

```diff
@@ -59,6 +60,17 @@ as well as the following technologies for its infrastructure:
 #### These things depend upon this solution
 - Consumer Applications (MAPPS)
 
+## Environments
+
+This API is designed to run in 4 environments:
+
+| Name           | Deployed | Data                                                | Purpose                                           |
+|----------------|----------|------------------------------------------------|--------------------------------------------------------|
+| Local          | N/A      | Prism Stub                                         | Local Feature Development                          |
+| Development    | On Merge  | T3/Development Seed                                | Internal Testing                                  |
```

Comment:
I'm worry that the person reading this document won't know what T3 is?

Reply:
Yeah a very cryptic name. The `.../Development` was meant to clear that up.